src/core/issues-notes.js

```diff
@@ -146,6 +160,35 @@ function handleIssues($ins, ghIssues, conf) {
   }
 }
 
+//derives the text-color-class based on the illumination score of the labelColor
+function deriveTextColorClass(hexColor) {
+  const rgb = parseInt(hexColor, 16);
+  if (isNaN(rgb)) {
+    throw new TypeError(`Invalid hex color: ${hexColor}`);
+  }
+  const red = (rgb >> 16) & 0xff;
+  const green = (rgb >> 8) & 0xff;
+  const blue = (rgb >> 0) & 0xff;
+  const illumination = 0.2126 * red + 0.7152 * green + 0.0722 * blue;
+  return illumination > 140 ? "light" : "dark";
+}
+
+function createLabels(label) {
+  const { color, url, name } = label;
```

Comment:
Sorry, I misunderstood what `url` was for... that's not the URL we want :) we need to construct a different one above.

Reply:
Yep